MekHQ/.classpath

```diff
@@ -20,5 +20,6 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
 	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="lib" path="lib/commonmark-0.13.0.jar"/>
```

Comment:
@neoancient would know how to integrate this with our Gradle setup.

Reply:
and with 3f024d8, I have changed things to do it the gradle way.